meshpy/geometry_set.py

```diff
@@ -246,6 +246,10 @@ def get_all_nodes(self):
                 "Currently GeometrySet are only implemented for points and lines"
             )
 
+    def get_geometry_objects(self):
+        """Return the objects of the specified geometry type."""
+        return self.geometry_objects[self.geometry_type]
```

Comment:
This returns a `dict`, because that is the way we store the data internally due to the ordered set issue discussed in https://github.com/imcs-compsim/meshpy/pull/138. But wouldn't it make more sense to return `list(self.geometry_objects[self.geometry_type].keys())` here? As a user I have no interest in the dictionary, only in a list containing the added elements in the correct order.

Reply:
Yes! This makes sense totally sense, especially since at every point I have converted them afterwards, any time.
Thanks for the hint!